haxepunk/graphics/BitmapText.hx

```diff
@@ -254,14 +227,9 @@ class BitmapText extends Graphic
 	function parseText():Void
 	{
 		// clear current opcode list
-		while (opCodes.length > 0)
-		{
-			opCodes.pop();
-		}
-		while (_scaleStack.length > 0)
-		{
-			_scaleStack.pop();
-		}
+		HXP.clear(opCodes);
```

Comment:
I don't see where it is creating a new array in hxcpp. Specifically I'm looking at this file. https://github.com/HaxeFoundation/hxcpp/blob/master/src/Array.cpp#L238

For other targets it sets the length to zero which truncates the array. I feel like I must be missing something...